### ISSUE TITLE
Introduction to Genetic Algorithm

### ISSUE THREAD

**Issue by arjasa1103** [arjasa1103/IntelligentSystemResources#1]
## TL;DR
Genetic algorithm use fitness function to determine fitness score and select individuals with highest fitness score to reproduce and create a better offspring that solves the problem

### Article Link
[Introduction to genetic algorithms including example code](https://towardsdatascience.com/introduction-to-genetic-algorithms-including-example-code-e396e98d8bf3)

### Author
Vijini Mallawaarachchi

## Key Takeaways
* Fitness function to determine fitness score
* Crossover to produce new offspring by exchanging genes
* Mutation to maintain diversity and prevent premature convergence

## Useful Code Snippets
```python
START
Generate the initial population
Compute fitness
REPEAT
    Selection
    Crossover
    Mutation
    Compute fitness
UNTIL population has converged
STOP
```
